src/classes/CRLP_Query_SEL.cls

```diff
@@ -210,20 +208,27 @@ public inherited sharing class CRLP_Query_SEL {
             allFieldsToQuery.add(SObjectType.Opportunity.fields.Amount.getSobjectField().getDescribe());
         }
 
+        Set<String> queryFields = new Set<String>();
```

Comment:
I think we talked through this to mutual satisfaction, but for a hypothetical future refactor it might be nice to pass the Set<String> of fields through all the methods that add to this query just as the most robust way to ensure absolutely uniqueness of every field in the list, regardless of the source (CC: @lmeerkatz) However I think that's beyond the scope for this fix.